### PR TITLE
Add `activeColorTheme` API

### PR DESCRIPTION
**Pull request by DucNgn**
Signed-off-by: DukeNgn <duc.a.nguyen@ericsson.com>

#### Review checklist

- [ ] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/plugin-ext/src/plugin/theming-ext.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    private ondDidChangeActiveColorTheme: Emitter<ColorTheme>;
+
+    constructor(
+        readonly rpc: RPCProtocol
+    ) {
+        this.ondDidChangeActiveColorTheme = new Emitter<ColorTheme>();
+    }
+
+    public get activeColorTheme(): ColorTheme {
```
Comment (paul-marechal) [inline packages/plugin-ext/src/plugin/theming-ext.ts]: `public` is already implicit.

File: packages/plugin-ext/src/plugin/theming-ext.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                kind = ColorThemeKind.Dark;
+                break;
+            case 'hc':
+                kind = ColorThemeKind.HighContrast;
+                break;
+        }
+        return kind;
+    }
+
+    public get onDidChangeActiveColorTheme(): Event<ColorTheme> {
```
Comment (paul-marechal) [inline packages/plugin-ext/src/plugin/theming-ext.ts]: Maybe better to hoist this with the other getter, and leave `public` implicit.

**Review by paul-marechal — COMMENTED** [review]
LGTM

**Review by paul-marechal — APPROVED** [review]

**Comment by DucNgn** [comment]
@marechal-p Thanks for the review 👍
I'm closing this one since I moved this PR to upstream Theia.

**Pull request closed without merging**